pkg/osquery/tables/execparsers/dpkg/parser.go

```diff
@@ -0,0 +1,47 @@
+package dpkg
+
+import (
+	"bufio"
+	//"fmt"
+	"io"
+	"strings"
+
+	"golang.org/x/exp/slices"
+)
+
+var allowedKeys = []string{
+	"package",
+	"essential",
+	"priority",
+	"section",
+	"version",
+	"description",
+	"build_essential",
+}
+
+func dpkgParse(reader io.Reader) (any, error) {
+	results := make([]map[string]string, 0)
+	row := make(map[string]string)
+
+	scanner := bufio.NewScanner(reader)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if len(line) == 0 {
+			results = append(results, row)
```

Comment:
Could you add a comment saying that a empty newline means we're done processing data for the current package, so add that row and make a new one for the next package?

Reply:
Added